studio/styles/code.scss

```diff
@@ -30,46 +35,45 @@
   .token.doc-comment.parameter.punctuation,
   .token.doc-comment.optional-parameter.punctuation {
     font-style: normal !important;
-    color: #ccc !important;
+    @apply text-gray-600 dark:text-gray-300 #{!important};
   }
 
   .token.keyword,
   .token.doc-comment.keyword {
     font-style: normal !important;
-    color: #569cd6 !important;
+    @apply text-blue-600 dark:text-blue-300 #{!important};
   }
 
   .token.string {
     font-style: normal !important;
-    color: #24b47e !important;
+    @apply text-green-500 #{!important};
   }
 
   .token.console {
```

Comment:
i think we can remove this class actually. Seems like there's no token called `console` from prism
https://prismjs.com/tokens.html

then we can remove the teal class in the tailwind.config as well

@MildTomato - just to confirm that i'm not missing anything?